cmd/multiFaAcceleration/multiFaAcceleration.go

```diff
@@ -0,0 +1,504 @@
+// Command Group: "Statistics & Population Genetics"
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/numbers"
+	"log"
+	"math"
+)
+
+//Settings contains all program arguments and options, and is passed throughout helper functions to aid readability.
+type Settings struct {
+	InFile                     string
+	ChromName                  string
+	VelOut                     string
+	AccelOut                   string
+	InitialVelOut              string
+	SearchSpaceBed             string
+	SearchSpaceProportion      float64
+	WindowSize                 int
+	UseSnpDistance             bool
+	Verbose                    bool
+	Epsilon                    float64
+	AllowNegative              bool
+	ZeroDistanceWeightConstant float64
+}
+
+//Once we have branch lengths for each valid window, we will need to normalize the values relative to each other. Thus, we store the branch lengths in this intermediate cache before writing to file.
+type BranchCache struct {
+	ChromStart int
+	ChromEnd   int
+	b1         float64
+	b3         float64
+}
+
+//A set of all observed pairwise distances between the four species. While these numbers must be integers, we store them as float64 to avoid casting in other functions as a way to improve readability.
+type Distances struct {
+	D01 float64
+	D02 float64
+	D03 float64
+	D12 float64
+	D13 float64
+	D23 float64
+}
+
+//The set of branch lengths corresponding to a particular distance matrix.
+type BranchLengths struct {
```

Comment:
I'm realizing it is a bit confusing to conceptualize the additions behind the distance calculations (D01, etc) because the distances are zero based and the branches are one based. Have you thought about having everything zero based for consistency?